azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,118 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and supported resources.
+    """
+
+    helps["dt data-history create adx"] = """
+        type: command
+        short-summary: Creates a data history connection between a Digital Twins instance and an Azure
+            Data Explorer Cluster. Requires pre-created Azure Data Explorer and Event Hub resources.
```

Comment:
nit, think Cluster should be lower case ....similar treatment here https://docs.microsoft.com/en-us/azure/data-explorer/create-cluster-database-portal?tabs=one-click-create-database